src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -203,10 +203,39 @@ public function renderListElement(
             $environment
         );
 
+        if (\is_object($listElement)) {
```

Comment:
Can you please extract this part into a new method

Reply:
Done